chain/client/src/test_utils/mock_resharding_sender.rs

```diff
@@ -0,0 +1,16 @@
+use std::collections::VecDeque;
+use std::sync::{Arc, RwLock};
+
+use near_async::messaging::CanSend;
+use near_chain::resharding::types::FlatStorageSplitShardRequest;
+
+#[derive(Clone, Default)]
+pub struct MockReshardingSender {
+    split_shard_request: Arc<RwLock<VecDeque<FlatStorageSplitShardRequest>>>,
+}
+
+impl CanSend<FlatStorageSplitShardRequest> for MockReshardingSender {
```

Comment:
This is the old framework for tests, before test loop. Let's try to keep it as lean as possible and not introduce any new functionality here? It would be really really really really really really great it we can just have a noop sender and kill all this code and just say we don't support resharding for old framework.

Reply:
I'll try to kill this code and replace with the least amount of code just to make stuff compile